### PR TITLE
Style fixes to fix EA specific header problems

### PR DESCRIPTION
**Pull request by jpaddison3**

### CODE REVIEW

**Comment by jpaddison3** [comment]
This was problematic
image

File: packages/lesswrong/components/common/Header.jsx [hunk 1]
```diff
@@ -230,12 +233,12 @@ class Header extends Component {
...
         >
           <AppBar className={classes.appBar} position="static" color={theme.palette.headerType || "default"}>
-            <Toolbar>
+            <Toolbar disableGutters>
               {this.renderNavigationMenuButton()}
               <Typography className={classes.title} variant="title" color="textSecondary">
                 <Hidden smDown implementation="css">
                   <Link to="/" className={classes.titleLink}>
-                    <Components.SiteLogo/>
+                    <div className={classes.siteLogo}><Components.SiteLogo/></div>
```
Comment (colophonemes) [inline packages/lesswrong/components/common/Header.jsx]: Should this be applied to the logo component itself, not the wrapper?

File: packages/lesswrong/components/common/Header.jsx [hunk 2]
```diff
@@ -246,13 +249,13 @@ class Header extends Component {
                 </Hidden>
                 <Hidden mdUp implementation="css">
                   <Link to="/" className={classes.titleLink}>
-                    <Components.SiteLogo/>
+                    <div className={classes.siteLogo}><Components.SiteLogo/></div>
```
Comment (colophonemes) [inline packages/lesswrong/components/common/Header.jsx]: As above

File: packages/lesswrong/components/common/Home2.jsx [hunk 3]
```diff
@@ -28,18 +29,7 @@ class Home2 extends PureComponent {
...
-                <AddBoxIcon />
-                New Shortform Post
-              </SectionButton>
-            </div>}
-          </SectionTitle>
-          {showShortformFeed && <CommentsNewForm
-              post={{_id:shortformFeedId}}
-              type="comment"
-            />}
+          <SectionTitle title="Recent Discussion" />
```
Comment (colophonemes) [inline packages/lesswrong/components/common/Home2.jsx]: Is this EA Forum specific?

File: packages/lesswrong/components/common/Header.jsx (hunk 1 above)
Comment (jpaddison3) [reply packages/lesswrong/components/common/Header.jsx]: The component, as is common, does not accept classname as a prop. Although I could refactor the component, I think it makes sense to generally just wrap things in divs. Could be wrong, I don't trust my taste much in this area yet.

File: packages/lesswrong/components/common/Home2.jsx (hunk 3 above)
Comment (jpaddison3) [reply packages/lesswrong/components/common/Home2.jsx]: No, except in as much as I think that was a merge mistake.

**Pull request merged**